recipes/eglibc/eglibc_2.18-r24943.oe.sig

```diff
@@ -1 +1,2 @@
+517cfd34b8d250dcdca0f3490fa4e9bf571ee473  svn://www.eglibc.org/branches/eglibc-2_18;rev=r24943
```

Comment:
Based on the svn log information, I would say that the checkout with svn protocol (ie. with this PR) is correct, and the old result was wrong.